collectfast/storage_extensions/gcloud.py

```diff
@@ -0,0 +1,26 @@
+import base64
+import binascii
+
+from google.api_core.exceptions import NotFound
+
+from collectfast.storage_extensions.base import BaseStorageExtensions
+
+
+class GoogleCloudStorageExtensions(BaseStorageExtensions):
+    """
+    Storage extensions for django-storage's `GoogleCloudStorage`
+    """
+
+    def get_etag(self, path):
+        normalized_path = path.replace('\\', '/')
+        try:
+            md5_base64 = self.storage.bucket.get_blob(normalized_path)._properties['md5Hash']
```

Comment:
Did you verify that the `get_blob()` call only fetches metadata and not the actual blob here?